### ISSUE TITLE
Failed to migrate SQLite to MySql 

### ISSUE THREAD

**Issue by AndreiKolosov** [tractr/directus-sync#41]
Hi! I have a Directus instance on SQLite3. When I try to migrate to a MySql instance with the --force flag, I get the following error.
Both instances are version 10.9.3.
![image](https://github.com/tractr/directus-sync/assets/91629845/c929f386-f825-4200-a09a-03a496ab1584)

**Comment by EdouardDem** [comment]
@AndreiKolosov The [collection service](https://github.com/directus/directus/blob/8336b4d868625c65e018160a3a24cde868aa9452/api/src/services/collections.ts#L62) does not allow to create a collection that start with `directus_`.
In your case, this is the `directus_sync_id_map`collection.
This collection is created by the [directus-extension-sync](https://www.npmjs.com/package/directus-extension-sync). This extension should be install in your new Directus instance (the one with MySQL) before using the [directus-sync CLI](https://www.npmjs.com/package/directus-sync).
Did you install this extension ?

**Comment by AndreiKolosov** [comment]
@EdouardDem Yes, I installed these dependencies in both projects. 
![image](https://github.com/tractr/directus-sync/assets/91629845/9754e511-57ed-4954-8c6f-b379f9284296)

**Comment by AndreiKolosov** [comment]
@EdouardDem From the above, I understood that the `directus_sync_id_map` collection is created at the time of installing the additional extension. I decided to conduct an experiment. I dropped the database, reinstalled all the dependencies on the MySQl instance and pushed again. Everything worked out. Apparently, for some reason I did not have a collection that was essential for proper operation. Thanks for the help!
